single-line-diagram/single-line-diagram-cgmes/single-line-diagram-cgmes-layout/src/main/java/com/powsybl/sld/cgmes/layout/LayoutToCgmesExtensionsConverter.java

```diff
@@ -241,6 +210,41 @@ private LayoutInfo applyLayout(Network network, String substationId, double xoff
         return subsBoundary;
     }
 
+    private void applyLayoutOnLines(VoltageLevelGraph vlGraph, VoltageLevel voltageLevel, String diagramName, OffsetPoint offsetPoint) {
```

Comment:
Could it be `applyLayoutOnLinesAndDanglingLines()` instead ? It is a bit long but it may be clearer that we are dealing with lines (classic lines) and dangling lines here.

Reply:
Rather create two methods: one `applyLayoutOnLines()` and one `applyLayoutOnDanglingLines()`, there's no benefit of doing it together. `isLineNode()` can thus be deleted (this shows that dealing with vsc converter station was maybe forgotten here...)